python/llm/src/bigdl/llm/transformers/convert_ipex.py

```diff
@@ -177,126 +189,3 @@ def _make_causal_mask(
         mask.masked_fill_(context_mask.bool(), torch.finfo(dtype).min)
 
     return mask[None, None, :, :].expand(bsz, 1, tgt_len, tgt_len + past_key_values_length)
-
-
-from transformers.modeling_outputs import BaseModelOutputWithPast
-from transformers.modeling_attn_mask_utils import _prepare_4d_causal_attention_mask
-
-
-def _llama_model_forward_4_35(
```

Comment:
If this function is removed, `https://github.com/huggingface/transformers/blob/v4.35.2/src/transformers/models/llama/modeling_llama.py` can be removed from the license of this file